boards/arm/ubx_evkninab4_nrf52833/ubx_evkninab4_nrf52833.dts

```diff
@@ -0,0 +1,231 @@
+		};
+		button1: button_1 {
+			/* EVK-NINA-B4 SW2 button */
+			gpios = <&gpio0 2 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button switch 1";
+		};
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 4 0>,	/* A0 */
+			   <1 0 &gpio0 30 0>,	/* A1 */
+			   <2 0 &gpio0 5 0>,	/* A2 */
+			   <3 0 &gpio0 2 0>,	/* A3 */
+			   <4 0 &gpio0 28 0>,	/* A4 */
+			   <5 0 &gpio0 3 0>,	/* A5 */
+			   <6 0 &gpio0 29 0>,	/* D0 */
+			   <7 0 &gpio1 5 0>,	/* D1 */
+			   <8 0 &gpio0 23 0>,	/* D2 */
+			   <9 0 &gpio0 31 0>,	/* D3 */
+			   <10 0 &gpio0 13 0>,	/* D4 */
+			   <11 0 &gpio0 11 0>,	/* D5 */
+			   <12 0 &gpio0 9 0>,	/* D6 */ /* NFC use by default */
+			   <13 0 &gpio0 10 0>,	/* D7 */ /* NFC use by default */
+			   <14 0 &gpio1 9 0>,	/* D8 */
+			   <15 0 &gpio0 12 0>,	/* D9 */
+			   <16 0 &gpio0 0 0>,	/* D10 */ /* Disconnected, see EVK User Guide */
+			   <17 0 &gpio0 1 0>,	/* D11 */ /* Disconnected, see EVK User Guide */
+			   <18 0 &gpio1 0 0>,	/* D12 */
+			   <19 0 &gpio0 7 0>,	/* D13 */
+			   <20 0 &gpio0 16 0>,	/* D14 */  /* SDA */
+			   <21 0 &gpio0 17 0>;	/* D15 */  /* SCL */
+	};
+
+	arduino_adc: analog-connector {
+		compatible = "arduino,uno-adc";
+		#io-channel-cells = <1>;
+		io-channel-map = <0 &adc 2>,	/* A0 = P0.04 = AIN2 */
+				 <1 &adc 6>,	/* A1 = P0.30 = AIN6 */
+				 <2 &adc 3>,	/* A2 = P0.5 = AIN3 */
+				 <3 &adc 0>,	/* A3 = P0.2 = AIN0 */
+				 <4 &adc 4>,	/* A4 = P0.28 = AIN4 */
+				 <5 &adc 1>;	/* A5 = P0.3 = AIN1 */
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		pwm-led0 = &red_pwm_led;
+		pwm-led1 = &green_pwm_led;
+		pwm-led2 = &blue_pwm_led;
+		red-pwm-led = &red_pwm_led;
+		green-pwm-led = &green_pwm_led;
+		blue-pwm-led = &blue_pwm_led;
+		sw0 = &button0;
+		sw1 = &button1;
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <37>;     /* P1.05 */
+	rx-pin = <29>;
+	rts-pin = <31>;
+	cts-pin = <23>;
+};
+
+arduino_i2c: &i2c0 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	sda-pin = <16>;
+	scl-pin = <17>;
+};
+
+&spi0 {
+	compatible = "nordic,nrf-spi";
+	/* Cannot be used together with i2c0. */
+	/* status = "okay"; */
+	sck-pin = <7>;
+	mosi-pin = <32>;
+	miso-pin = <15>;
+};
+
+&spi1 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	sck-pin = <3>;
+	mosi-pin = <28>;
+	miso-pin = <9>;
+};
+
+&pwm0 {
+	status = "okay";
+	ch0-pin = <33>;
+	ch0-inverted;
+	status = "okay";
+	ch1-pin = <13>;
+	ch1-inverted;
+	status = "okay";
+	ch2-pin = <32>;
+	ch2-inverted;
+};
+
+arduino_spi: &spi0 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
```

Comment:
Sorry, I missed that previously. This should be `spi2` or `spi3` (`"nordic,nrf-spim"` in this case), as `spi0` is already set earlier, so another modification here would be confusing (and it also cannot be used together with `i2c0`).

Reply:
Thanks - I changed it to `spi2` and `nordic,nrf-spim`